docs/Configuring-Orion/Using-Oracle.md

```diff
@@ -0,0 +1,59 @@
+description: Storing Information with Oracle
+<!--- END of page meta data -->
+
+# Store Information with Oracle
+
+Oracle is an object-relational database you can use to store private transaction payloads for Orion. Use a relational database to support your [disaster recovery strategy](Disaster-Recovery.md).
+
+## Prerequisites
+
+* [Oracle Database](https://docs.oracle.com/en/)
+
+## Set up your Oracle database
+
+In your database:
+
+1. Create the `store` table by using the provided DDL script, as shown in the following example. Substitute your own values for `HOST-NAME`, `USER-NAME`, and `DATABASE-NAME`.
+
+    `sqlplus -h <HOST-NAME> -U <USER-NAME> -d <DATABASE-NAME> -f ~<INSTALLATION_DIRECTORY>/orion/database/oracle_ddl.sql` 
+    
+2. Specify the users for the `store` table and assign permissions to them.
+
+## Configure Orion to use Oracle
+
+In your [Orion configuration file](Configuration-File.md), specify `oracle` for the `storage` property, as shown in the following example. Substitute your own values for `USER-NAME`, `PASSWORD`, `HOST`, `PORT`, and `SID`. `SID` identifies a specific database on a system. 
```

Comment:
identifies a specific database on a system -> identifies a specific Oracle database

Suggested change to make more specific (_system_ is one of those red flag words to me)

Reply:
Changed.